python/pyspark/worker.py

```diff
@@ -86,21 +86,29 @@ def wrap_udf(f, return_type):
         return lambda *a: f(*a)
 
 
-def wrap_scalar_pandas_udf(f, return_type):
+def verify_scalar_pandas_udf_result_length(result, length):
```

Comment:
This method is not called outside `wrap_scalar_pandas_udf`. Could you define it inside and rename it to `verify_result_length`?